content/en/synthetics/private_locations/dimensioning.md

```diff
@@ -68,7 +68,9 @@ For example, Datadog recommends ~ 8 cores CPU, ~ 10GiB memory, and ~ 5GiB disk f
 
 When multiple workers are associated with a private location, each worker requests a couple test runs, which depend on the [`concurrency` parameter][4] and on the number of additional test runs that can be assigned.   
 
-For example, ten tests are scheduled to run simultaneously on a private location that has two workers running. If Worker 1 is running two tests, Worker 1 can request three additional tests to run. If Worker 2 is not running any tests, Worker 2 can request the five following tests. The remaining two tests can be requested by whichever worker has finished running its test first (any worker that has available slots).
+For example, ten tests are scheduled to run simultaneously on a private location that has two workers running. If Worker 1 is running two tests, Worker 1 can request three additional tests to run. If Worker 2 is not running any tests, Worker 2 can request the five following tests. 
+
+The remaining three tests can be requested by whichever worker has finished running its test first (any worker that has available slots).
```

Comment:
I updated this from two to three, feel free to change it back if my math is wrong!

Reply:
I think `2` was correct since from the 10 tests: one worker requests 3, the other requests 5 so 2 are left

Reply:
```suggestion
The remaining two tests can be requested by whichever worker has finished running its test first (any worker that has available slots).
```